modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -59,9 +63,20 @@ object Main
           cfg.snapshot
         )
 
+        _ <- GlobalSnapshotEventsPublisher
+          .publishEvents(queues.stateChannelOutput, snapshotTriggerStream, services.gossip)
+          .asResource
+
+        globalSnapshotConsensus <- GlobalSnapshotConsensus
```

Comment:
same here, it could go to services or programs, because indeed a consensus is kinda service

Reply:
I wouldn't qualify it as a service yet, as it doesn't contain any methods to call, but in the future, it may become a service, as we may add some methods that will get called when we trigger/complete redownload. All in all, I think it won't hurt if I will add it to services already.